README.md

```diff
@@ -1,37 +1,155 @@
 
+## Installation
+
+Building TLF requires some other components to be installed first. 
+It depends on:
+
+* the `hamlib` library for controlling your radio,
+* `ncurses` and `tinfo` for text screen handling,
+* the `XMLRPC_C` library for communication with programs like `Fldigi`
+* `Glib-2.0` for supporting C functions.
+
+If you are using a distribution of the Debian family (including Ubuntu) you can get all of these dependencies by opening a Terminal window and typing:
+
+```
+sudo apt install libglib2.0-dev libhamlib-dev libncurses5-dev libtinfo-dev libxmlrpc-core-c3-dev
+```
+
+While not strict build dependencies, the following packages may be helpful too:
+
+* `sox` for audio signal handling during SSB contests and
+* `xplanet` which allows you to see the latest DX spots on the globe.
+
+On Debian Linuxes:
+
+```
+sudo apt install sox xplanet
+```
+
+Once the dependencies are installed, the easiest way to install tlf is by downloading the latest tarball (version 1.4.1) from [here](http://download.savannah.gnu.org/releases/tlf/tlf-1.4.1.tar.gz), then navigating your Terminal to the directory where you unpacked it, and typing:
+
+```
+./configure
+make
+sudo make install
+```
+
+If you plan to do contests in digimodes using Fldigi, start the sequence above with
+
+```
+./configure --enable-fldigi-xmlrpc
+```
+
+followed by the `make` and `make install` commands.
+
+If you are compiling tlf from a clone of the Github repository instead of the tarball, please do
+
+```
+autoreconf --install
+```
+
+before the above commands.
+
+By default, TLF will install into `/usr/local/bin`, together with the
+shell scripts from the scripts directory. Data files will install into
+`/usr/local/share/tlf`.
+
+If you want to change any of these paths, you can do that with the
+configure script too, e.g. 
+
+```
+./configure --prefix=/usr --datadir=/usr/share
+```
+
+will install TLF and scripts into `/usr/bin`, datafiles into `/usr/share/tlf`.
+
+If you want to use [cwdaemon](https://github.com/acerion/cwdaemon) for CW, make sure it's installed, and either set it up to start at system startup, or plan to start it manually before launching TLF. 
+
+If you want to use a K1EL Winkeyer for CW, download and install either [winkeyer_server](https://github.com/ok2cqr/winkeyer_server) from OK2CQR or N0NB's [winkeydaemon](https://github.com/N0NB/winkeydaemon), following the instructions on those pages. You'll need to start the Winkeyer server (either one) before starting TLF. Both programs work by impersonating cwdaemon, which TLF will use automatically if you've enabled the NETKEYER settings in the `logcfg.dat` file.
```

Comment:
While I appreciate the mention, I did not write `winkeydaemon`.  It has many authors of which I only contributed to a very minor degree.  I did collate the patches I could find and give the project a home in order that it could stay current.  Simply mentioning `winkeydaemon` without my callsign as possessive is preferred.